### PR TITLE
My Courses Page

### PR DESCRIPTION
**Pull request by sakshi-choudhary**
**ISSUE** #159

**Describe the changes you've made**
Created a My Courses page and added a list of all the courses bought by the student, and after clicking on a particular course, the video and chapters of the course page show up.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**
NA

**Checklist**

- [x] My code follows the code style of this project.
- [x] The title of my pull request is a short description of the requested changes.

**Preview**

and after clicking on any course card their particular course video and chapters page opens

### CODE REVIEW

**Comment by sakshi-choudhary** [comment]
Please review @Abhishek-kumar09

File: src/Routes.js [hunk 1]
```diff
@@ -20,7 +21,8 @@ const renderRoutes = () => (
       <MainLayout>
         <SwipeableRoutes replace>
           <Route path="/home" exact component={HomePage} />
-          <Route path="/personal" exact component={CoursePage} />
+          <Route path="/mycourses" exact component={MyCourses} />
+          <Route path="/mycourses/each" exact component={CoursePage} />
```
Comment (Abhishek-kumar09) [inline src/Routes.js:25]: `mycourses/:coursename` make param instead of definite route

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Review by Abhijay007 — COMMENTED** [review]
@sakshi-choudhary I reviewed your code and there are some minor changes I want to suggest:

bottom nav

please change "My classes" to "My courses" in the bottom nav and change the icon for the same

And please implement a back-to-top button at the bottom of the page

**Comment by sakshi-choudhary** [comment]
I have made the requested changes @Abhishek-kumar09 @Abhijay007, Please do a review.

**Comment by Abhijay007** [comment]
Routing-issue

@sakshi-choudhary I think there is an issue in routing the `wishlist` section is not shows up until we select a course from the `Myclass` section. I am not sure about it, can you please cross-check it and confirm it once. I made a video in gif format of the problem. As you can see in the video this wishlist is section is replaced by the profile section but when we selected a course from " MyClass" it works normally.

**Comment by sakshi-choudhary** [comment]
> ssue in routing the `wishlist` sectio

Yes, thanks for letting me know, that was a bug. I have resolved it. @Abhijay007

**Comment by sakshi-choudhary** [comment]
Ready for review @Abhishek-kumar09

**Pull request merged**